frontend/src/components/personal/EmailChangeForm.tsx

```diff
@@ -21,9 +21,9 @@ export default function EmailChangeForm (): React.ReactElement {
       email: Yup.string()
       .required('Enter your new Email')
       .email('This is not email')
+      .notOneOf([(user.email)], 'New email must differ')
```

Comment:
:+1:

Reply:
Dont understand by now how to compare this is all validation is in separate file..